src/index.html

```diff
@@ -17,11 +21,401 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
```

Comment:
The href attribute should point to a CSS file, not a SCSS file. Make sure to include the compiled CSS file in your HTML.